src/components/HeroList.js

```diff
@@ -0,0 +1,38 @@
+import Hero from "./Hero";
+
+// 이미지가 존재하는지 확인합니다.
+const isImagePresent = (data) => {
+  const pathData = data.thumbnail.path.split("/");
+  if (pathData[pathData.length - 1] === "image_not_available") {
+    return false;
+  }
+  return true;
+};
```

Comment:
```suggestion
const isImagePresent = (data) => {
  const pathData = data.thumbnail.path.split("/");

  return pathData[pathData.length - 1] !== "image_not_available";
};
```

`===` 에서 `!==` 로만 바꾼 뒤에 바로 반환해줘도 좋을 것 같아요!